roles/chameleon_image_tools/tasks/main.yml

```diff
@@ -6,6 +6,58 @@
     name: "{{ chameleon_image_tools_docker_image }}"
     force_source: yes
 
+# Cleanup old service, scripts, and timer (if present)
+- name: Cleanup old image_deploy timer (stop & disable)
+  become: yes
+  systemd:
+    name: image_deploy.timer
+    state: stopped
+    enabled: no
+  ignore_errors: yes
+  when: enable_image_deployer | bool
+
+- name: Cleanup old image_deploy service (stop & disable)
+  become: yes
+  systemd:
+    name: image_deploy.service
+    state: stopped
+    enabled: no
+  ignore_errors: yes
+  when: enable_image_deployer | bool
+
+- name: Remove old timer unit file
+  become: yes
+  file:
+    path: /etc/systemd/system/image_deploy.timer
+    state: absent
+  ignore_errors: yes
+  when: enable_image_deployer | bool
+
+- name: Remove old service unit file
+  become: yes
+  file:
+    path: /etc/systemd/system/image_deploy.service
+    state: absent
+  ignore_errors: yes
+  when: enable_image_deployer | bool
+
+- name: Remove old chameleon_image_tools config dir
+  become: yes
+  file:
+    path: /etc/chameleon_image_tools
+    state: absent
+  ignore_errors: yes
+  when: enable_image_deployer | bool
+
+- name: Remove old wrapper script
+  become: yes
+  file:
+    path: /usr/local/sbin/chameleon-image-deploy
+    state: absent
+  ignore_errors: yes
+  when: enable_image_deployer | bool
```

Comment:
these should be moved to a block or separate task file

and, instead of ignoring errors, which still print in red, we should have a step which checks for the presence of the old systemd units, and IFF that check finds them, runs the removal tasks